vector/src/main/java/im/vector/app/features/createdirect/CreateDirectRoomActivity.kt

```diff
@@ -100,7 +100,7 @@ class CreateDirectRoomActivity : SimpleFragmentActivity(), UserListViewModel.Fac
                             title = getString(R.string.fab_menu_create_chat),
                             menuResId = R.menu.vector_create_direct_room,
                             singleSelection = true,
-                            showContactBookAction = false,
+                            showContactBookAction = true,
```

Comment:
Can you please revert this change? the request #248 has been cancelled for the moment